File: src/question.ts
function getQuestionParticle(c: string, softOffset: number): string {
    return chooseMBP(c, softOffset, MAME, BABE, PAPE);
}

function getColloquialQuestionParticle(c: string, softOffset: number): string {
    return chooseMBP(c, softOffset, CLQ_MAME, CLQ_BABE, CLQ_PAPE);
}

File: kotlin/grammar/src/main/kotlin/com/khairulin/kazakhverb/grammar/Question.kt
package com.khairulin.kazakhverb.grammar

object Question {
    fun getQuestionParticle(char: Char, softOffset: Int): String {
        return AffixHelpers.chooseMBP(char, softOffset, Rules.MAME, Rules.BABE, Rules.PAPE)
    }
}